public/app/features/explore/spec/helper/interactions.ts

```diff
@@ -33,6 +33,28 @@ export const openQueryHistory = async (exploreId: ExploreId = ExploreId.left) =>
   ).toBeInTheDocument();
 };
 
+export const closeQueryHistory = async (exploreId: ExploreId = ExploreId.left) => {
+  const closeButton = withinExplore(exploreId).getByRole('button', { name: 'Close query history' });
+  userEvent.click(closeButton);
+};
+
+export const switchToQueryHistoryTab = async (
+  name: 'Settings' | 'Query History',
+  exploreId: ExploreId = ExploreId.left
+) => {
+  userEvent.click(withinExplore(exploreId).getByLabelText(`Tab ${name}`));
```

Comment:
we can query by the tab role + its accessible name rather than just by its label text:
```suggestion
  userEvent.click(withinExplore(exploreId).getByRole('tab', { name: `Tab ${name}` }));
```

Reply:
👍 updated in 5be9baa21e7b517b2e29f8af9dbec4320db405d8